exporters/zipkin/src/test/java/io/opentelemetry/exporter/zipkin/ZipkinTestSpan.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.zipkin;
+
+import static org.mockito.Mockito.mock;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.TraceFlags;
+import io.opentelemetry.api.trace.TraceState;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.testing.trace.TestSpanData;
+import io.opentelemetry.sdk.trace.data.EventData;
+import io.opentelemetry.sdk.trace.data.StatusData;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+import java.net.InetAddress;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import zipkin2.Endpoint;
+import zipkin2.Span;
+
+class ZipkinTestSpan {
```

Comment:
The name of this class is confusing since its not actual a span but a collection of help methods to create spans. How about something like `ZipkinTestUtil`?

Reply:
Yeah good call. It made more sense originally when the it was just `ZipkinTestSpan.build()` but other stuff got lumped in and it doesn't make nearly as much sense now.